Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py

```diff
@@ -388,7 +388,7 @@ def install(self):
                 else:
                     # We might not need setup.py at all, check if we have dist-info and the library in the temporary location
                     to_be_moved = os.listdir(temp_location)
-                    has_module_or_package = self.name in to_be_moved or '{}.py'.format(self.name) in to_be_moved
+                    has_module_or_package = self.name.split('.')[0] in to_be_moved or '{}.py'.format(self.name) in to_be_moved
```

Comment:
With meta path finders, I think the latter (module) case matters too—see, for example, `six.moves`. It would be odd for someone to register `six.moves` as an AutoInstaller package, but… there might be some other case where it does make sense?